reports/push_notifications/push_user_er.sql

```diff
@@ -0,0 +1,63 @@
+WITH sample AS (
+   SELECT client_id,
+          date_parse(submission_date_s3, '%Y%m%d') as normalized_submission_date,
+          popup_notification_stats['web-notifications'].action_1 AS push_acceptance,
+          web_notification_shown
+   FROM main_summary
+   WHERE submission_date_s3 > date_format(CURRENT_DATE - interval '6' MONTH, '%Y%m%d')
+     AND submission_date_s3 < date_format(CURRENT_DATE, '%Y%m%d')
+     AND sample_id='42'
+),
+push_accepters AS (
+   SELECT DISTINCT client_id,
+                   normalized_submission_date AS eligibility_start,
+                   normalized_submission_date + interval '30' DAY AS eligibility_end
```

Comment:
What about using a window up to the current date so that we get rid of false negatives?